### PR TITLE
TextEditor , cloudinary 

### PR DESCRIPTION
**Pull request by ahmedmsayeem**
changes

1. setup Basic RichTextEditor using react-quill 
          - has preview option on various devices
          - connected to one of the events in Db
          - checkout: [https://flc-website-git-dev-9964728107s-projects.vercel.app/textEditor ](https://flc-website-git-dev-9964728107s-projects.vercel.app/textEditor)

2.  cloudinary (SignIn before test)
          - change in prop names and comments to make usage intuitive
          - removed unnecesary mutation from delete componet, to only support deleting images from cloudinary ,to allow for all 
            usecases          
          - [https://flc-website-git-dev-9964728107s-projects.vercel.app/cloudinary](https://flc-website-git-dev-9964728107s-projects.vercel.app/cloudinary)

3. payment button - [https://flc-website-git-dev-9964728107s-projects.vercel.app/payment](https://flc-website-git-dev-9964728107s-projects.vercel.app/payment)

### CODE REVIEW

File: src/zod/attendenceZ.ts
Comment (satwikrprabhu) [inline src/zod/attendenceZ.ts:1]: this already exists as toggleTeamAttendanceZ in the same file. Please cross check it once

Comment (satwikrprabhu) [reply src/zod/attendenceZ.ts:1]: Ohh wait this file already exists. There is a typo in the filename

Comment (ahmedmsayeem) [reply src/zod/attendenceZ.ts:1]: yes vro...there were 2 files in the codebase when i synched my fork
1.attendance
2.attendance

i uncommented one which was giving ts errors
didnt add or delete code in it

what shall i do

Comment (nandanpi) [reply src/zod/attendenceZ.ts:1]: @ahmedmsayeem This PR has a lot of commits that I had removed when I rebased the repo.
I'll close this now. Please cleanup your git log, fix the commits and make a fresh New PR and request my review

**Pull request closed without merging**